src/hipscat/io/write_metadata.py

```diff
@@ -79,7 +81,7 @@ def write_legacy_metadata(args, histogram, pixel_map):
         histogram (:obj:`np.array`): one-dimensional numpy array of long integers where the
             value at each index corresponds to the number of objects found at the healpix pixel.
         pixel_map (:obj:`np.array`): one-dimensional numpy array of integer 3-tuples.
-            See `pixel_math.partition_stats.generate_alignment` for more details on this format.
+            See :func:`~hipscat.pixel_math.partition_stats.generate_alignment` for more details on this format.
```

Comment:
Will this `:func:` do some automagical hyper linking if the docstring is run through Sphinx?

Reply:
no need for the `:func:` part, but yes, the full namespaces are required to be properly picked up by sphinx...

Reply:
Drew - yes - the `:func:` gets picked up by sphinx apidoc. I wasn't able to get the cross-linking to show up without it, however.

Reply:
the full namespace will be needed in almost all cases. When in doubt, you can double-check `objects.inv`. (also note, in some cases sphinx is misbehaving when building docs on osx due to the OS's case insensitivity)